GitUI/CommandsDialogs/SettingsDialog/Pages/FormBrowseRepoSettingsPage.cs

```diff
@@ -1,10 +1,23 @@
 ﻿using GitCommands;
+using GitUI.Hotkey;
 using GitUI.Shells;
+using GitUIPluginInterfaces;
+using ResourceManager;
+using ResourceManager.Hotkey;
 
 namespace GitUI.CommandsDialogs.SettingsDialog.Pages
 {
     public partial class FormBrowseRepoSettingsPage : SettingsPageWithHeader
     {
+        private const string _processHistoryUrl = "https://git-extensions-documentation.readthedocs.io/settings.html#process-history-as-tab-otherwise-as-panel";
+        private readonly TranslationString _processHistoryTooltip
+            = new("""
+                  The output displayed in the process dialog and the trace output is retained and shown in the Process History.
+                  Focus the Process History or toggle its visibility using the hotkey {0}.
+                  - With this set, the Process History is displayed in a tab in the lower pane of the Browse Repository window.
+                  - With this unset, the Process History is displayed in a panel docked to the lower left corner of the Browse Repository window.
+                  The Process History can be disabled by setting the process history depth to 0.
```

Comment:
This tooltip is a little hard to read, so suggestion to:
- add empty lines to make easier to read
- Put to "focus/hide", which is not "really" linked to the setting explanation at the end
- reword "set" to "check"

I also propose to remove the disable part to put it in the label (as already said in another comment) to make this tooltip a little more easier to read.

```suggestion
                  The output displayed in the process dialog and the trace output is retained and shown in the Process History.

                  - With this checked, the Process History is displayed as a tab in the lower pane of the Browse Repository window.
                  - With this unchecked, the Process History is displayed in a panel docked to the lower left corner of the Browse Repository window.

                  The Process History can be disabled by setting the process history depth to 0.
                  Focus the Process History or toggle its visibility using the hotkey {0}.
```

Reply:
done but without
> reword "set" to "check"

because this wording is used in other places, too

---

I am going to rename the classes later.